arviz/data/io_cmdstan.py

```diff
@@ -7,15 +7,15 @@
 import logging
 import re
 
-
 import numpy as np
 import pandas as pd
 import xarray as xr
+import arviz.utils as utils
```

Comment:
One more system import :) Since you're importing utils below you should be able to delete this one